SonarQube.Common/FilePropertiesProvider.cs

```diff
@@ -24,11 +24,6 @@ public class FilePropertiesProvider : ISonarPropertyProvider
         /// </summary>
         private IDictionary<string, string> properties;
 
-        /// <summary>
-        /// Property name comparisons are be case-sensitive
-        /// </summary>
-        private static StringComparer PropertyKeyComparer = StringComparer.InvariantCulture;
```

Comment:
On second thought - the key comparer is probably ok as Ordinal, the value comparer you might want to set to the CurrentCulture